packages/@okta/i18n/src/properties/login.properties

```diff
@@ -448,6 +448,16 @@ verify.webauthn.biometric.instructions=Your browser or device will prompt you to
 webauthn.biometric.error.factorNotSupported = Security key or biometric authenticator is not supported on this browser. Select another factor or contact your admin for assistance.
 webauthn.biometric.error.factorNotSupported.oneFactor = Security key or biometric authenticator is not supported on this browser. Contact your admin for assistance.
 enroll.webauthn.instructions.noSupportForBiometric = Note: Some browsers may not support biometric authenticators.
+authfactor.webauthn.error.clientdata_parsing_failed=Client data parsing failed
+authfactor.webauthn.error.clientdata_message_type_mismatch=Message type in client data doesn't match expected message type.
+authfactor.webauthn.error.clientdata_challenge_missing=No challenge parameter in client data
+authfactor.webauthn.error.clientdata_challenge_mismatch=Challenge in client data doesn't match with server side value
+authfactor.webauthn.error.clientdata_origin_missing=No origin parameter in client data
+authfactor.webauthn.error.clientdata_origin_mismatch=Origin parameter in client data doesnt match. clientData origin was {0}
```

Comment:
`authfactor.webauthn.error.clientdata_origin_mismatch=Origin parameter in client data doesn't match. clientData origin was {0}`

Small update -- I forgot an apostrophe in my PR so this should be the above.